exporter/metric/metric.go

```diff
@@ -74,6 +74,48 @@ type metricExporter struct {
 	startTime time.Time
 }
 
+
+// TODO: put all constants into a separate file in the same package
+
+// Mappings for the well-known OpenCensus resource label keys
+// to applicable Stackdriver Monitored Resource label keys.
+// A uniquely identifying name for the Kubernetes cluster. Kubernetes
+// does not have cluster names as an internal concept so this may be
+// set to any meaningful value within the environment. For example,
+// GKE clusters have a name which can be used for this label.
+const (
+	CloudKeyZone         = "cloud.zone"
+
+	K8SType              = "k8s"
+	K8SKeyClusterName    = "k8s.cluster.name"
+	K8SKeyNamespaceName  = "k8s.namespace.name"
+	K8SKeyPodName        = "k8s.pod.name"
+	K8SKeyDeploymentName = "k8s.deployment.name"
+
+	stackdriverGenericTaskNamespace = "contrib.opencensus.io/exporter/stackdriver/generic_task/namespace"
+	stackdriverGenericTaskJob       = "contrib.opencensus.io/exporter/stackdriver/generic_task/job"
+	stackdriverGenericTaskID        = "contrib.opencensus.io/exporter/stackdriver/generic_task/task_id"
+)
+
+var k8sContainerMap = map[string]string{
+//	"project_id":     "",
```

Comment:
I think we do not need to care about the field project_id in the OT resources because we would set the project_id  to the value of the project_id of the metrics exporter option in the resourceToMonitoredResourcepb function.